packages/playwright-core/src/cli/program.ts

```diff
@@ -362,7 +362,7 @@ async function launchContext(options: Options, headless: boolean, executablePath
 
   // In headful mode, use host device scale factor for things to look nice.
   // In headless, keep things the way it works in Playwright by default.
-  // Assume high-dpi on MacOS. TODO: this is not perfect.
+  // Assume high-dpi on macOS. TODO: this is not perfect.
```

Comment:
highdpi, hidpi, high-dpi are all used in this repository. I don't have a rule that complains about them yet but I do notice when the two hyphenless forms are both used (neither was used enough times to justify suggesting changing the rest to match, but I didn't notice `high-dpi` until now).